plugins/gazebo_gui/GazeboGui.cc

```diff
@@ -175,26 +138,12 @@ bool GazeboGui::Load(const tinyxml2::XMLElement *_elem)
         << "missing the filename attribute. Skipping this plugin.\n";
       continue;
     }
-
-    ignition::gui::App()->LoadPlugin(file, elem);
+    app->LoadPlugin(file, elem);
   }
 
-  std::string worldName = "default";
-
-  // Get the world name
-  elem = _elem->FirstChildElement("world_name");
```

Comment:
I think this line can be removed:

https://github.com/ignitionrobotics/ign-launch/blob/1e9bdf6d0b21e024ae65ffda55644644ac22db52/examples/gazebo_plugins.ign#L103

Reply:
Removed.